<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>todo.txt Plasmoid Config Dialog</string>
  </property>
  <widget class="QLabel" name="FileLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>25</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLabel" name="ColourLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Priority Colours</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>181</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::ExpandingFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="priorityALabel">
      <property name="text">
       <string>Priority A</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="priorityBLabel">
      <property name="text">
       <string>Priority B</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="KColorButton" name="priorityBColour"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="priorityCLabel">
      <property name="text">
       <string>Priority C</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="KColorButton" name="priorityCColour"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="priorityDLabel">
      <property name="text">
       <string>Priority D</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="KColorButton" name="priorityDColour"/>
    </item>
    <item row="0" column="1">
     <widget class="KColorButton" name="priorityAColour"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>31</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="todoFileLabel">
      <property name="text">
       <string>todo.txt location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KUrlComboRequester" name="todoURL"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>todoURL</tabstop>
  <tabstop>priorityAColour</tabstop>
  <tabstop>priorityBColour</tabstop>
  <tabstop>priorityCColour</tabstop>
  <tabstop>priorityDColour</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>todoURL</sender>
   <signal>openFileDialog(KUrlRequester*)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priorityAColour</sender>
   <signal>changed(QColor)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priorityBColour</sender>
   <signal>changed(QColor)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priorityCColour</sender>
   <signal>changed(QColor)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priorityDColour</sender>
   <signal>changed(QColor)</signal>
   <receiver>ConfigDialog</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
